simulator/src/resim/cmd_show_ledger.rs

```diff
@@ -13,7 +13,7 @@ pub struct ShowLedger {}
 
 impl ShowLedger {
     pub fn run<O: std::io::Write>(&self, out: &mut O) -> Result<(), Error> {
-        let ledger = RadixEngineDB::with_bootstrap(get_data_dir()?);
```

Comment:
If we no longer do (optional) bootstrapping in all these commands, how is a user supposed to get started / bootstrapped? Does this make using resim harder?

Reply:
Thought empty ledger state dump should be okay, but maybe it's more user friendly if we always bootstrap for any resim commands that read/write the substate store. I will revert.